### PR TITLE
Add gh ct lint validation

### PR DESCRIPTION
**Pull request by guangbochen**
Add gh ct lint validation

### CODE REVIEW

File: .github/workflows/release.yaml [hunk 1]
```diff
@@ -1,25 +1,64 @@
...
+    branches:
+      - master
   push:
     branches:
       - master
+      - release*
 
 jobs:
-  release:
+  lint-test:
```
Comment (yaocw2020) [inline .github/workflows/release.yaml:13]: How about separating into three jobs: lint, test and release?  And then we can define the dependencies among the jobs with `jobs.<job_id>.needs`. Refer to https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#jobs.

File: tests/ct.yaml [hunk 2]
```diff
@@ -0,0 +1,4 @@
+remote: upstream
+target-branch: master
+chart-dirs:
+  - charts
```
Comment (yaocw2020) [inline tests/ct.yaml:4]: What's the purpose of this file?

File: .github/workflows/release.yaml (hunk 1 above)
Comment (guangbochen) [reply .github/workflows/release.yaml]: updated

File: tests/ct.yaml (hunk 2 above)
Comment (guangbochen) [reply tests/ct.yaml:4]: this is the ct lint config file https://github.com/helm/chart-testing#config-file

**Review by gitlawr — APPROVED** [review]
LGTM

**Pull request merged**